src/main.js

```diff
@@ -205,13 +206,42 @@ function testForDuplicate(arrayOfPosters, mainPoster) {
 function saveMainPoster() {
   var mainPoster = new Poster(image.src, title.innerText, quote.innerText);
   if(!testForDuplicate(savedPosters, mainPoster)) {
-  savedPosters.push(mainPoster);
-  savedPostersGrid.innerHTML += `
-  <section class=mini-poster>
-    <img class="poster-img" src="${mainPoster.imageURL}">
-    <h1 class="poster-title">${mainPoster.title}</h1>
-    <h3 class="poster-quote">${mainPoster.quote}</h3>
-  </section>
+  savedPosters.unshift(mainPoster);
+  displaySavedPosters();
+  var allMiniPosters = document.querySelectorAll('.mini-poster')
+  for (var i = 0; i < allMiniPosters.length; i++) {
+    // struggling to figure out event listener syntax: how do we add an argument to the event handler?
+    allMiniPosters[i].addEventListener('dblclick', function() {
+      savedPosters.splice(i, 1)
+      displaySavedPosters()
+    })
+  }
```

Comment:
So this is tricky, and not in a way that is easily figured out. The tl;dr is that adding event listeners within a for loop is not reliable because of a scoping problem (apologies if you haven't had a lesson on scope yet, you'll learn more about it later). For now, the short of it is that the `i` in your `eventListener` callback will always equal the length of your for loop. 

Here is a really good article on that problem: https://gomakethings.com/why-you-shouldnt-attach-event-listeners-in-a-for-loop-with-vanilla-javascript/

--- 

So maybe we should approach this the opposite way.

```
  // 1. If a user clicks on any poster we should run a function
  // 2. We should loop through all our savedPosters
  // 3. During each loop we should be checking if this savedPoster is the one we clicked
  // 4. If it is, remove it
```

Now, as far as point 3 is concerned, you might be wondering how we can identify which poster we clicked on from the eventListener, as indicated by your `how do we add an argument` comment. 

Event Listeners do get access to an `event` object in the callback method, and those have a ton of cool properties, but maybe primary among them is `event.target`, which is the DOM element that received the event. 

So I would suggest setting up an event listener on the entire saved posters area, and when a poster is clicked on you should identify the event target, then see if there is something that exists on the event target that would help you identify it while looping through the `savedPosters`. Then when that is identified, remove it from the list. 

--- 

Here are some MDN pages I would suggest looking at for more information:
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Usage_notes
https://developer.mozilla.org/en-US/docs/Web/API/Event/target

---

This is a lot of information, and I hope I have summed it up well enough. I would be happy to pair with you all once you've given it a shot. I am available at 3pm tomorrow, or later tomorrow night.